tree/dataframe/inc/ROOT/RDFInterface.hxx

```diff
@@ -519,7 +519,7 @@ public:
          snapCall << RDFInternal::ColumnName2ColumnTypeName(c, nsID, tree, fDataSource, isCustom) << ", ";
       };
       if (!columnList.empty())
-         snapCall.seekp(-2, snapCall.cur); // remove the last ",
+         snapCall.seekp(-2, snapCall.cur);                          // remove the last ",
```

Comment:
@imaxoi please revert this change, clang-format is not always right unfortunately